src/sources/journald.rs

```diff
@@ -191,9 +190,19 @@ impl JournaldConfig {
         matches
     }
 
+    /// Builds the `schema::Definition` for this source using the provided `LogNamespace`.
     fn schema_definition(&self, log_namespace: LogNamespace) -> Definition {
-        BytesDeserializerConfig
-            .schema_definition(log_namespace)
+        let definition = Definition::new_with_default_metadata(
+            Kind::object(
+                Collection::empty()
+                    .with_unknown(Kind::bytes())
+                    // the MESSAGE field is most likely present in the journald Record, but may not be.
+                    .with_known("message", Kind::bytes().or_undefined()),
+            ),
+            [log_namespace],
+        );
+
+        definition
             .with_standard_vector_source_metadata()
             .with_source_metadata(
```

Comment:
I'd move these up to where you're creating the `Definition` - just slightly neater. (How you built in the tests)

Reply:
I'll move them since I don't have a strong preference.

I intentionally kept them separate in this instance because I though it was more clear what the "base" namespace definition is. When we're using a `Deserializer` , its not much code and thus pretty clear.
The tests I left because who cares.